ballerina/LICENSE

```diff
@@ -17,169 +17,170 @@ ________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2
 commons-text-1.10.0.jar                                                                             bundle         apache2
 compiler-0.8.9.jar                                                                                  jar            apache2
-ballerina-rt-2201.7.0.jar                                                                           jar            apache2
-typedesc-2201.7.0.jar                                                                               jar            apache2
-shell-core-2201.7.0.jar                                                                             jar            apache2
+testerina-runtime-2201.7.1.jar                                             jar            apache2
```

Comment:
Let's fix the indentation here and the other places

Reply:
Updated